src/background/alarms.ts

```diff
@@ -44,4 +53,9 @@ export const periodicUpdater = newPeriodicTask(
   dispatchUpdateTaskForAllAccountsIDs,
   1
 );
+
 export const periodicTaxiUpdater = newPeriodicTask(Alarm.TaxiUpdate, dispatchUpdateTaxiAction, 1);
+
+export const clearAllPeriodicUpdaters = () => {
```

Comment:
I am bit of nACK in stopping updading utxo set after logout, for reason stated in main comment